Components/kube-ui.css

```diff
@@ -0,0 +1,40 @@
+@import './avatar.css';
+@import './badge.css';
+
+/* import all other smaller components css here */
+html {
+	scroll-behavior: smooth;
+}
+
+*{
+	margin: 0;
+    padding: 0;
+	font-family: 'Poppins', sans-serif;
+}
+body {
+	color: #3e4152;
+}
+
+:root {
+	--primary-color: #7C0BC2;
+	/* --light-primary-color: #8829C2; not sure of color yet */
+	--dark-primary-color: #4F0D75;
+
+	--complimentary-text-color: white;
+
+	--secondary-text-color: #696b79;
+	--dark-secondary-text-color: #3f4049;
+	--secondary-color: #eaeaec;
+	--dark-secondary-color: #d8d8da;
+
+	--success-color: rgb(51, 145, 51);
+	--danger-color: rgb(235, 55, 55);
+
+	--inactive-color: rgb(191, 192, 198);
+	--z-index-1: 1;
+	--z-index-2: 2;
```

Comment:
Use the same types of color codes. If using Hex codes use it for all the colors.

Reply:
Changed rgb to hex value